Tutorials/intro-to-octave/configuration.py

```diff
@@ -5,27 +5,10 @@
 ######################
 qop_ip = "127.0.0.1"
 qop_port = 80
-
 ############################
 # Set octave configuration #
 ############################
 octave_1 = OctaveUnit("octave1", qop_ip, port=50, con="con1", clock="Internal")
```

Comment:
No need for clock anymore and default port is 11050:
`octave_1 = OctaveUnit("octave1", qop_ip, port=11050, con="con1")`